src/NHibernate/Driver/NpgsqlDriver.cs

```diff
@@ -24,8 +25,23 @@ namespace NHibernate.Driver
 	/// <a href="http://pgfoundry.org/projects/npgsql">http://pgfoundry.org/projects/npgsql</a>. 
 	/// </p>
 	/// </remarks>
+#if DRIVER_PACKAGE
+	public class NpgsqlDriver : DriverBase
```

Comment:
Could be renamed `PostgreSQLDriver` or `PostgresDriver`, if we accept abstracting away the chosen data provider.